src/main/java/com/commercetools/sync/inventories/utils/InventoryUpdateActionUtils.java

```diff
@@ -43,7 +42,7 @@ private InventoryUpdateActionUtils() {
                                                                                    @Nonnull final InventoryEntryDraft
                                                                                        newEntry) {
         final Long oldQuantityOnStock = oldEntry.getQuantityOnStock();
-        final Long newQuantityOnStock = newEntry.getQuantityOnStock() == null ? Long.valueOf(0L) : newEntry
+        final Long newQuantityOnStock = newEntry.getQuantityOnStock() == null ? NumberUtils.LONG_ZERO : newEntry
```

Comment:
Yes, but I am still not sure how is this an improvement, when the Long.valueOf(0L) has it's internal caching.